src/main/java/com/google/codeu/data/Tea.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright 2019 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.codeu.data;
+
+// import java.util.UUID;
+
+/** A single message posted by a user. */
+public class Tea {
```

Comment:
This solution seems overly complicated. How about:

```
public enum TeaCategory {
   GREEN,
   WHITE,
   BLACK,
   HERBAL;
}
```

And then use `Map<TeaCategory, Integer>` to represent a histogram of each category rather than maintaining separate integer variables for each and every category?